### DISCUSSION TITLE
Proxmox 7 over Debian install script

### DISCUSSION THREAD

**Discussion by potter-jason** [tteck/Proxmox#1264] [General]
Could you do a script to install Proxmox 7 over Debian?

Ala...

https://computingforgeeks.com/how-to-install-proxmox-ve-on-debian-bullseye/

thx!

**Comment by tteck** [comment]
What is the reason for choosing this option instead of https://www.proxmox.com/en/downloads/item/proxmox-ve-7-4-iso-installer?

**Comment by potter-jason** [comment]
I used that option on two of my proxmox servers. But I have a use case
where I need a debian instance beneath the proxmox one. This seems like its
supported, but fairly complicated to get set up, hence the reason i asked
for a script.

Thanks and great job. I've been going thru all your scripts, probably tried
each one at least once

On Wed, Mar 29, 2023 at 9:18 AM tteckster ***@***.***> wrote:

> What is the reason for choosing this option instead of
> https://www.proxmox.com/en/downloads/item/proxmox-ve-7-4-iso-installer?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tteck/Proxmox/discussions/1264#discussioncomment-5467994>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZP6C4MDSXJSQOZM3PBJYB3W6ROGJANCNFSM6AAAAAAWJER534>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Jason Potter
